db/migrate/20170707162731_add_columns_unit_time_to_showback_rate.rb

```diff
@@ -0,0 +1,10 @@
+class AddColumnsUnitTimeToShowbackRate < ActiveRecord::Migration[5.0]
+  def change
+    add_column :showback_rates, :fixed_rate_per_unit,    :string
+    add_column :showback_rates, :fixed_rate_per_time,    :string
+    add_column :showback_rates, :variable_rate_per_unit, :string
+    add_column :showback_rates, :variable_rate_per_time, :string
+    add_column :showback_rates, :step_value,             :float
+    add_column :showback_rates, :step_unit,              :string
```

Comment:
Add step_per_time_unit step_per_time_value to reflect that the minimum step can be 60 s  or  1h, or 60 m